bulletroboy/state_mapper/state_mapper.py

```diff
@@ -8,10 +8,10 @@
 
 from ..utils.utils import Topics, Services
 from roboy_simulation_msgs.msg import Collision
+from roboy_middleware_msgs.msg import EFPose
```

Comment:
EFPose is not a middleware message. it's a control message so needs to be moved. besides, it's not on the roboy3 or dashing branch. will not approve this PR until messages are merged